streamx-console/streamx-console-service/src/main/resources/db/migration/V1_0__init_db.sql

```diff
@@ -229,8 +229,10 @@ PRIMARY KEY (`ROLE_ID`) USING BTREE
 -- ----------------------------
 DROP TABLE IF EXISTS `t_role_menu`;
 CREATE TABLE `t_role_menu` (
+`ID` bigint NOT NULL AUTO_INCREMENT,
```

Comment:
I think the sql changes should be written to V1_2_1__upgrade, otherwise users who upgrade from 1.1.0 will encounter flyway validation errors.